### DISCUSSION TITLE
803

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#978] [Comments]
# 803

국가기술자격증 시험에는 재밌는 제도가 있다.평균 60점이 넘으면 시험을 통과할 수 있는데, 나머지 과목들을 모두 100점을 맞아도 한 과목이 40점 이하이면 과락으로 탈락이다.즉, 5과목 중 4과목이 100점이고 한 과목이 39점이면 평균 87.8점임에도 불합격이다.모든 과목을 60점이상 맞을 필요는 없지만, 한 과목이라도 40점 이하는 안된다는 이 제도가 스타트업 생활을 하면서 자주 생각하게 된다.작은 회사에서 큰 회사로 가는 경험을 계속 하다보면 "장점의 극대화 vs 단점의 최소화" 의 결정들을 자주 보게 된다.주변의 다른 사람들과 비교해 내가 갖고 있는 장점을 더 극대화 하는 것으로 커리어를 발전시켜나갈 것이냐, 아니면 타인에 비해 부족한 점을 채워나갈 것이냐인 것이다.예를 들어 아직 서비스가 일정..

https://jojoldu.tistory.com/803

**Comment by kein84** [comment]
ㅎㅎ 잘 읽었습니다. 요새는 자기 개발 서적이나 동기부여 서적에 나올 이야기를 많이 하시는군요

실무 관련 아티클도 올려주시면 더 좋을 거 같아요

항상 수고 많으십니다

**Reply by jojoldu** [reply]
네 ㅎㅎ 저는 항상 블로그에 지금 제가 풀어야할 고민과 문제들을 정리하고 있어서, 지금의 제가 풀어야할 문제도 이런 문제들이 많은것 같아요 :) 
회사일과 개인발전을 분리한다면 가능할것같긴한데, 
회사의 기술적 고민은 아무래도 사내 위키 등에 녹여낼 수 밖에 없는것 같아요ㅎㅎ;;

**Comment by knockknockyoo** [comment]
기술보다 내적성장을 고민하게 되신거 같네요. 이제 시니어 개발작의 여정이겠죠.

**Comment by pak0426** [comment]
게임에서 사기 캐릭처럼 육각형의 능력자가 되고 싶지만 현실은 그게 너무 힘든거 같아요. 그래도 스스로를 보듬어주고 항상 도전하는 개선하려는 모습이 멋진거 같아요.
